sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -216,6 +216,26 @@ def _batch_send(
         except StorageErrorException as error:
             process_storage_error(error)
 
+class TransportWrapper(HttpTransport):
```

Comment:
Could we please add a docstring to both this and the async wrapper class just to describe their purpose.